script/build.ts

```diff
@@ -55,15 +58,31 @@ updateLicenseDump(err => {
       process.exit(1)
     } else {
       console.log(`Built to ${appPaths}`)
-      process.exit(0)
     }
   })
 })
 
-function packageApp(callback) {
-  const options = {
-    name: distInfo.getExecutableName(),
-    platform: process.platform,
+function packageApp(
+  callback: (error: Error | null, appPaths: string | string[]) => void
+) {
+  // not sure if this is needed anywhere, so I'm just going to inline it here
+  // for now and see what the future brings...
+  const toPackagePlatform = (platform: NodeJS.Platform) => {
```

Comment:
Good spot! 

> I think this should be `platform === 'win32...'

I'm going to keep the checks around for `darwin` and `linux` because the [supported list](https://github.com/electron-userland/electron-packager/blob/master/docs/api.md#platform) of platforms is rather narrow.

> Allowed values: `linux`, `win32`, `darwin`, `mas`, `all`